tests/Tests.AzureAppConfiguration/Tests.cs

```diff
@@ -186,7 +186,7 @@ public void TestUserAgentHeader()
             // 4. Contains the name and version of the App Configuration SDK package
             // 5. Contains the runtime information (target framework, OS description etc.) in the format set by the SDK
             // 6. Does not contain any additional components
-            string userAgentRegex = @"^Microsoft\.Extensions\.Configuration\.AzureAppConfiguration/\d+\.\d+\.\d+(-preview)?(-\d+-\d+)?,azsdk-net-Data.AppConfiguration/[.+\w-]+ \([.;\w\s]+\)$";
+            string userAgentRegex = @"^Microsoft\.Extensions\.Configuration\.AzureAppConfiguration/\d+\.\d+\.\d+(-preview)?(\.\d+)?,azsdk-net-Data.AppConfiguration/[.+\w-]+ \([.;\w\s]+\)$";
```

Comment:
does this regex pass the test if version is "7.0.0.3"?

Reply:
I didn't realize that, but I moved the check to inside the check for `-preview` so should fix that